.travis.yml

```diff
@@ -10,17 +10,24 @@ script:
     else
       VERSION=$(git log --format=%H -1)
     fi
-  - sbt ++$TRAVIS_SCALA_VERSION 'set version in ThisBuild := "'$VERSION'"' test
+  - sbt ++${TRAVIS_SCALA_VERSION}! 'set version in ThisBuild := "'$VERSION'"' clean test
 after_success:
   - |
     if [ $TRAVIS_PULL_REQUEST == "false" ] && ([ "${TRAVIS_BRANCH:-}" == "master" ] || [ "${TRAVIS_TAG:-}" != "" ]); then
-      sbt ++$TRAVIS_SCALA_VERSION 'set version in ThisBuild := "'$VERSION'"' "set publishMavenStyle in ThisBuild := false" publish
-      # Keep publishing to the "generic" bintray repository
-      sbt ++$TRAVIS_SCALA_VERSION 'set version in ThisBuild := "'$VERSION'"' "set publishMavenStyle in ThisBuild := true" publish
-      # Publish to the "maven" bintray repository as well
-      sbt ++$TRAVIS_SCALA_VERSION \
+      export BINTRAY_REPO=${BINTRAY_REPO:-scala-hedgehog-maven}
+      sbt ++${TRAVIS_SCALA_VERSION}! \
         'set version in ThisBuild := "'$VERSION'"' \
         'set publishMavenStyle in ThisBuild := true' \
-        'set bintrayRepository in ThisBuild := "scala-hedgehog-maven"' \
+        'set bintrayRepository in ThisBuild := "'$BINTRAY_REPO'"' \
```

Comment:
Have you tested this? I found the problem was that `publishMavenStyle` didn't actually work.

Reply:
I think it's more likely that setting `BINTRAY_REPO` is what's really working here.

Reply:
> I think it's more likely that setting BINTRAY_REPO is what's really working here.

I should have just done that instead in the first place, it's probably enough to make it all work.

Reply:
It works. 🙂
https://bintray.com/kevinlee/maven/hedgehog-core/31f2de43da717a846c533b3b716ce459260cba8f#files/qa/hedgehog